scripts/diff_l10n_branches.py

```diff
@@ -78,18 +83,70 @@ def process_diff_status(diff_status, l_commit, r_commit, src_lang_path,
     status_letter = diff_status[0]
     filepath = diff_status[1]
 
+    size_xs = 10
+    size_s = 30
+    size_m = 100
```

Comment:
nit: How do authors know the size of each category?

Reply:
@kbhawkey Thank you for the comment !

I applied the same labeling convention with k/website repo. 
(https://github.com/kubernetes/website/labels?page=4&sort=name-asc)

size/XS : Denotes a PR that changes 0-9 lines, ignoring generated files.
size/S : Denotes a PR that changes **10**-29 lines, ignoring generated files.
size/M : Denotes a PR that changes **30**-99 lines, ignoring generated files.
size/L : Denotes a PR that changes **100**-499 lines, ignoring generated files.
size/XL : Denotes a PR that changes **500**-999 lines, ignoring generated files.
size/XXL : Denotes a PR that changes **1000**+ lines, ignoring generated files.

So, I think contributors for k/website repo can guess the size of each category :)

Reply:
:+1: